prody/measure/transform.py

```diff
@@ -71,7 +71,7 @@ def getMatrix(self):
         """Returns a copy of the 4x4 transformation matrix whose top left is
         rotation matrix and last column is translation vector."""
 
-        if self._matrix is not None:
+        if self is not None and self._matrix is not None:
```

Comment:
I don't think this line would be ever reached if `self` were somehow `None`, because `None` wouldn't be an instance of this class

Reply:
yeah. I’ll have to look into it again